AssemblyUtil.spec

```diff
@@ -26,6 +26,20 @@ module AssemblyUtil {
                 returns (FastaAssemblyFile file) authentication required;
 
 
+    typedef structure {
+        list<string> ref_lst;
+    } GetFASTAParams;
+
+     typedef structure {
+        list<string> FASTA;
```

Comment:
Wouldn't it be useful to return data about the fasta files here? I'm thinking types in particular. I'd like to know if one of my refs is a metagenome, one is a contigset, one a genome, etc.

Reply:
Another point is that since the input refs can contain sets, it'd be useful to know the parent ref for any refs here.

Reply:
I added type data to refs in the last push. Now the output structure is {ref: {'path': fastaPath}, 'type': objectType}.

When you say 'parent ref' you mean the ref for the full set object?

Reply:
Basically if the user is passing in ref X and you're giving them back refs Y1...YN it'd be good to know wher ref Yn came from.

Reply:
Added parent ref info into the final object dictionary for the AssemblySet and Metagenome objects.

Reply:
The spec doesn't show these changes...?

Reply:
Not sure the changes you're talking about above ever got pushed here